apps/src/applab/api.js

```diff
@@ -328,8 +328,8 @@ export function startWebRequest(url, func) {
   return Applab.executeCmd(null, 'startWebRequest', {url: url, func: func});
 }
 
-export function startWebRequestSync(url, headers, func) {
-  return Applab.executeCmd(null, 'startWebRequestSync', {url, func, headers});
+export function startWebRequestSync(url, func) {
+  return Applab.executeCmd(null, 'startWebRequestSync', {url, func});
```

Comment:
the other new synchronous commands execute the async function but use the `nativeIsAsync` dropletConfig property to make them blocking -- could we do the same here now that the sync/async command signatures are the same? (i.e., execute `startWebRequest` here and remove [the `startWebRequestSync` command from applab/commands.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/commands.js#L1707))